### PR TITLE
Logger implementation for tests

### PR DESCRIPTION
**Pull request by mattkae**
# Logger in `mir-ci` :pen: 

## Resources
- https://pytest-with-eric.com/pytest-best-practices/pytest-logging/

## What's new?
- We have a barebones logging config that works
- We can output to a file if we want

### CODE REVIEW

File: mir-ci/mir_ci/benchmarker.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import psutil
 import os
 import time
+import logging
```
Comment (Saviq) [inline mir-ci/mir_ci/benchmarker.py:10]:
```suggestion
import logging

logger = logging.getLogger()
```

File: mir-ci/mir_ci/benchmarker.py [hunk 2]
```diff
@@ -98,19 +99,19 @@ def _on_packet(self, packet: ProcessInfoFrame) -> None:
         cpu_time_microseconds_total = packet.cpu_time_microseconds_total
         current_memory_bytes = packet.current_memory_bytes
         if pid is None:
-            print("Frame is lacking pid")
+            logging.getLogger().error("Frame is lacking pid")
```
Comment (Saviq) [inline mir-ci/mir_ci/benchmarker.py:102]:
```suggestion
            logger.error("Frame is lacking pid")
```

And all the way down, of course.

**Review by Saviq — CHANGES_REQUESTED** [review]
I believe it's common to have a global `logger` object, rather than getting new one each time?

File: mir-ci/mir_ci/benchmarker.py (hunk 1 above)
Comment (mattkae) [reply mir-ci/mir_ci/benchmarker.py:10]: Would this be better?
```
logger = logging.getLogger(__name__) 
```

Comment (Saviq) [reply mir-ci/mir_ci/benchmarker.py:10]: Yup, even more common :)

Comment (mattkae) [reply mir-ci/mir_ci/benchmarker.py:10]: Cool :)

**Review by Saviq — APPROVED** [review]
Sures!

**Pull request merged**